### DISCUSSION TITLE
Combobox issue

### DISCUSSION THREAD

**Discussion by SOAPkelele** [tailwindlabs/headlessui#2249] [Help]
**What package within Headless UI are you using?**

@headlessui/react

**What version of that package are you using?**

v1.7.8

**What browser are you using?**

Chrome

**Reproduction URL**

https://github.com/SOAPkelele/combobox_issue
Installation instructions are in readme

**Describe your issue**

I used example from the docs with my setup. Evertything works fine on Mac and IOS, both mobile and desktop, but combobox autocomplete doens't work properly on Android in Chrome. When typing some characters and there is some suggestions, and I press enter - the focus leaves the field and suggestions are still open, autocomplete doesn't happen. 
However, https://headlessui.com/examples/react-combobox-hero this example works fine. I guess it maybe my setup which behaves differently on Android. I can't figure out what it is.

**Comment by RobinMalfait** [comment]
Hey! Thank you for your bug report!
Much appreciated! 🙏

That's odd, I can't seem to reproduce this 🤔
- Do you have any extensions installed on your Android phone in the Chrome browser?
- Can you reproduce it on another browser?

**Reply by SOAPkelele** [reply]
No, I don't have any extensions. I've tried to reproduce it in Samsung Internet browser, the issue still there.  

I think I found the problem. I use Google keyboard and I tried to switch to the default one by Samsung, and it helped. Autocomplete now works. 
https://www.youtube.com/watch?v=znIOkyhVz5s here is a video to make it more clear
